sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCopilotPromoOptInStatus.md

```diff
@@ -0,0 +1,51 @@
+Get-SPOCopilotPromoOptInStatus 
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Get-SPOCopilotPromoOptInStatus
+applicable: SharePoint Online
+title: Get-SPOCopilotPromoOptInStatus.md
+schema: 
+author: swathi.iruvanti
+ms.author: siruvanti
+ms.reviewer:
+---
+# Get-SPOCopilotPromoOptInStatus
+
+## SYNOPSIS
+
+Returns the true or false Opt-In Copilot promo status 
+
+## SYNTAX
+```powershell
+Get-SPOCopilotPromoOptInStatus
+```
+
+## DESCRIPTION
+
+The `Get-SPOCopilotPromoOptInStatus` cmdlet retrieves and returns the opt-in Copilot promo status. The user must be a SharePoint Admin to run the cmlets.
```

Comment:
reword it to something like this:

```
This cmdlet returns the Opt-In Copilot promo status for the tenant.
If the promo status is enabled, the return value is `True`, otherwise the return value is `False`.
```